### DISCUSSION TITLE
Log volume histogram support total value caculate in frontend

### DISCUSSION THREAD

**Discussion by hanjm** [grafana/grafana#66200] [Observability]
Currently, the log volume histogram currently show the level count of every level in every interval, it can be caculated a total value in frontend and display it. it is useful to get the total amount of query result in full time-range.

<img width="1801" alt="image" src="https://user-images.githubusercontent.com/7621241/230752002-76992ba8-f89a-49e2-a9a0-ca18aa57a750.png">
<img width="1783" alt="image" src="https://user-images.githubusercontent.com/7621241/230752023-81df8d10-3f1d-4234-83b2-d7d66d5521ee.png">

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82706. Feel free to continue the discussion around this there. Thank you!
